python_modules/dagit/dagit/pipeline_execution_manager.py

```diff
@@ -93,6 +100,9 @@ def execute_pipeline(self, repository_container, pipeline, pipeline_run, throw_o
                     executor_config=InProcessExecutorConfig(
                         throw_on_user_error=throw_on_user_error
                     ),
+                    reexecution_config=pipeline_run.reexecution_config,
+                    step_keys_to_execute=pipeline_run.step_keys_to_execute,
+                    storage_mode=RunStorageMode.FILESYSTEM,
```

Comment:
[1] seems odd to me that we are hardcoding the storage mode here like this, if this is just the default dagit assumes might be worth doing `DAGIT_DEFAULT_STORAGE_MODE` const or something

Reply:
agree. it's actually worse than that because we don't honor if the storage mode is specified in the environment dict